data/cube/row/BaseRow.js

```diff
@@ -124,12 +124,10 @@ export class BaseRow {
             if (canAggregate[name]) {
                 const oldValue = data[name],
                     newValue = field.aggregator.replace(children, oldValue, update);
-                if (oldValue !== newValue) {
-                    update.oldValue = oldValue;
-                    update.newValue = newValue;
-                    myUpdates.push(update);
-                    data[name] = newValue;
-                }
+                update.oldValue = oldValue;
+                update.newValue = newValue;
+                myUpdates.push(update);
+                data[name] = newValue;
```

Comment:
I really don't have this all loaded in my head, but reading the description of the issue, I'm wondering if it is still a problem in any way that this still only happens within the `if (canAggregate[name]) {}` conditional.

Apologies, but I would need to be led through the whole thing again for me to properly say I've reviewed and understood this.